src/test/kotlin/com/atlassian/performance/tools/jiraactions/api/measure/ActionMeterTest.kt

```diff
@@ -98,6 +104,34 @@ class ActionMeterTest {
             .isNotNull
     }
 
+    @Test
+    fun shouldDrilldownConditionWork() {
+        val tick = ofSeconds(1)
+        val clock = TickingClock(start, tick)
+        val output = CollectionActionMetricOutput(mutableListOf())
+        val w3cPerformanceTimelineMock = HardcodedTimeline(RecordedPerformanceEntries(emptyList(), emptyList()))
+        val actionMeter = ActionMeter.Builder(
+            output = output
+        )
+            .drilldownCondition(Predicate { actionMetric -> CREATE_ISSUE.label == actionMetric.label })
```

Comment:
Nice test, clean and simple.
Can we make it a red test first? We can add the new API, but the SUT can ignore it (not call `predicate.test`) - that should make it red

- [ ] red-green pls

Reply:
I don't have plans to make it red-green. I'd just wanna finish the PR to unlock the work I wanted to work on.

Reply:
Everyone is always in a hurry. And then we complain about the quality, when we're in a hurry again in the future.
I'll do it

Reply:
You can `git reset --hard` onto https://github.com/dagguh/jira-actions/tree/issue/JCES-1334-drilldown-condition

Reply:
So the first commit adds the functionality with a bug, and another fixes it. I would not say it increases the quality.

Reply:
BTW working on a PR in parallel doesn't bring much value. The time needed to resolve conflicts probably would be greater than the help with the change.

Reply:
Ok, so adding a new feature and test for it in red-green fashion apparently "feels weird". Maybe so.
I won't push it further. I'll try to do the TDD red-first even for new features myself. Maybe it'll be more convincing.